src/python_testing/TC_RVCCLEANM_1_2.py

```diff
@@ -57,11 +57,16 @@ def pics_TC_RVCCLEANM_1_2(self) -> list[str]:
     async def test_TC_RVCCLEANM_1_2(self):
         self.endpoint = self.matter_test_config.endpoint
 
-        attributes = Clusters.RvcCleanMode.Attributes
+        RVCClean_cluster = Clusters.RvcCleanMode
+        attributes = RVCClean_cluster.Attributes
+        RVCClean_attr_list = attributes.AttributeList
+        attribute_list = await self.read_single_attribute_check_success(endpoint=self.endpoint, cluster=RVCClean_cluster, attribute=RVCClean_attr_list)
+        supported_modes_attr_id = attributes.SupportedModes.attribute_id
+        current_mode_attr_id = attributes.CurrentMode.attribute_id
 
         self.print_step(1, "Commissioning, already done")
 
-        if self.check_pics("RVCCLEANM.S.A0000"):
+        if supported_modes_attr_id in attribute_list:
```

Comment:
Q for the RVC folks - this is mandatory, right? Can we just remove this check?

Reply:
Yes it is. Same of `A0001`. Should this check be made somewhere else, or is it dealt by the data model?